apps/studio/styles/main.scss

```diff
@@ -2,10 +2,11 @@
 @tailwind components;
 @tailwind utilities;
 
-@import '~ui/build/css/source/global.css';
-@import '~ui/build/css/themes/dark.css';
-@import '~ui/build/css/themes/light.css';
-@import '~ui/build/css/themes/classic-dark.css';
+@import './../../../packages/ui/build/css/source/global.css';
```

Comment:
how long did this take to realize?

Reply:
haha, it was easy, it said import can't be resolved.